.github/PULL_REQUEST_TEMPLATE/other.md

```diff
@@ -0,0 +1,9 @@
+---
+name: Other
+about: Infra or meta change not falling in another category
```

Comment:
I don't think it's wise to limit the “other” category.
```suggestion
about: Any change not falling in another category
```

Reply:
Yeah perhaps not, I didn't have it limited originally, but my concern was that users (especially those who would be least familiar with the things mentioned in the checklist) would be more likely to immediately go for a generic "Other" rather than "Changing a PEP" if it wasn't explicitly qualified like this, and on this repo, any valid change that isn't "Changing a PEP" is, by definition, a change to the infrastructure (rendering system, workflows, makefile, config, etc) and/or a meta change, so I'm not sure what (acceptable) PR could plausibly not fall under any of those categories.

Reply:
Marking as resolved to be able to merge. We can adjust this later if needed.